examples/fault-injection/envoy.yaml

```diff
@@ -0,0 +1,55 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 9211
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          codec_type: auto
+          stat_prefix: ingress_http
+          access_log:
+            name: envoy.file_access_log
+            config:
+              path: /dev/stdout
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: service
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: /
+                route:
+                  cluster: local_service
+          http_filters:
+          - name: envoy.fault
+            config:
+              abort:
+                http_status: 503
+                percentage:
+                  numerator: 0
+                  denominator: HUNDRED
+          - name: envoy.router
+            config: {}
+  clusters:
+  - name: local_service
+    connect_timeout: 1s
+    type: strict_dns
+    lb_policy: round_robin
+    hosts:
```

Comment:
nit: can we use `load_assignment` instead of `hosts`? Since it is [deprecated](https://github.com/envoyproxy/envoy/blob/ddb3d63a8c10af4fe49716dc973400bc8c38c90d/api/envoy/api/v2/cds.proto#L169). (But feel free to leave it as it is for now, since I'll probably sweep all example to use load_assignment due to: https://github.com/envoyproxy/envoy/issues/4618). 🙏